fix: 채팅방 비밀번호 추가/변경/삭제를 바로 변경

peas856 opened the pull request:
## 개요
- #298 해결
- 유킴님 PwdSetModal.tsx 리코일 사용한 부분 맞는지 확인부탁드려요
- 현재 본인은 실시간 연동되나 비밀변호 변경, 삭제, 제거 후 다른 유저가 참여가능한 방 리스트 목록에서 실시간 연동은 안 될 것으로 예상 -> 하지만 이를 해결하기는 사실상 불가능하고 다시 참여가능한 방 리스트 버튼을 누르면 리프레쉬 되기때문에 괜찮을 것으로 판단
<!--
이 PR이 해결한 이슈를 아래처럼 추가해주세요.
resolved와 같은 키워드를 **반드시** 포함해야 이 PR이 머지될 때 연관된 이슈도 같이 닫힙니다.
참고: https://bumkeyy.gitbook.io/bumkeyy-code/project-management/pull-request

- closed #15
- resolved #16
-->

Commit:
feat: 머지 과정 충돌 에러 해결

Commit:
feat: 접속 페이지 디자인 Intra 로그인 할때만 표시, 첫 화면 메인페이지 id ===0일때 문구 변경

Commit:
feat: 비밀번호 수정 화면 실시간 연동

Commit:
feat: commit

scarf005 commented on frontend/app/src/view/PwdSetModal.tsx:
```diff
@@ -34,13 +36,15 @@ export const PwdSetModal = ({ open, off, socket }: Props) => {
       password: inputAdd.current?.value,
       command: 'ADD',
     })
+    setSelectedChat({ bool: true, roomId: roomId, roomType: 'PROTECTED' })
```
```suggestion
    setSelectedChat({ bool: true, roomId, roomType: 'PROTECTED' })
```

scarf005 commented on frontend/app/src/view/PwdSetModal.tsx:
```diff
@@ -34,13 +36,15 @@ export const PwdSetModal = ({ open, off, socket }: Props) => {
       roomId,
       command: 'DELETE',
     })
+    setSelectedChat({ bool: true, roomId: roomId, roomType: 'PUBLIC' })
```
```suggestion
    setSelectedChat({ bool: true, roomId, roomType: 'PUBLIC' })
```

scarf005 reviewed (approved):
확인했습니다

Commit:
- 소켓 이벤트 useChatSocket으로 옮김
- CHATUSER_STATUS 이벤트를 받으면 roomId의 해당 유저들 리스트만 업데이트 했으나 recoil로 접근시 roomId가 0이 찍혀(이유모름) 그냥 chat을 전부 인벨리데이트함 -> 비효율적임 수정 필요

peas856 committed:
Update frontend/app/src/view/PwdSetModal.tsx

Co-authored-by: scarf <greenscarf005@gmail.com>

peas856 committed:
Update frontend/app/src/view/PwdSetModal.tsx

Co-authored-by: scarf <greenscarf005@gmail.com>
